src/Dialect/Krnl/Krnl.td

```diff
@@ -500,6 +500,23 @@ def KrnlUnrollOp : Op<Krnl_Dialect, "unroll"> {
   }];
 }
 
+def KrnlParallelOp : Op<Krnl_Dialect, "parallel"> {
+  let summary = "Krnl parallel operation";
+  let description = [{
+    Parallelize the specified loops.
+    ```
+    krnl.parallel %i
```

Comment:
I believe you have constrains... e.g. it has to be the last operation before the krill.iterate, isn't it? If so, I would mention this here. For example, we don't want to parallelize and the block that loop. Or permute?